plutus-tx/src/PlutusTx/TH.hs

```diff
@@ -29,7 +29,7 @@ loadFromFile fp = TH.liftSplice $ do
     -- We don't have a 'Lift' instance for 'CompiledCode' (we could but it would be tedious),
     -- so we lift the bytestring and construct the value in the quote.
     bs <- liftIO $ BS.readFile fp
-    TH.examineSplice [|| SerializedCode bs Nothing mempty ||]
+    TH.examineSplice [|| SerializedCode bs Nothing Nothing ||]
```

Comment:
I this point I am wondering if there are any reasons not to make `SerializedCode` a record with fields and use them to increase readability?